web/package/agama-web-ui.changes

```diff
@@ -1,3 +1,11 @@
+-------------------------------------------------------------------
+Mon May 12 12:47:46 UTC 2025 - Ladislav Slezák <lslezak@suse.com>
+
+- Fixed React router navigation to avoid quickly switching between
```

Comment:
NP: I'd say "Fixed navigation to..." or "Fixed redirection to".

It was not a problem in React Router nor how we deal with it, but an Agama "state" problem and navigation performed at App.tsx.